src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,115 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    static final int DEFAULT_CAPACITY = 16;
+    static final float LOAD_FACTOR = 0.75f;
+
+    private int capasity;
+    private int size;
+    private Node<K,V>[] table;
+
+    public MyHashMap() {
+        capasity = DEFAULT_CAPACITY;
+        size = 0;
+        table = new Node[capasity];
+    }
+
+    static class Node<K,V> {
+        private final K key;
+        private V value;
+        private Node<K,V> next;
+
+        Node(K key, V value,Node<K,V> next) {
+            this.key = key;
+            this.value = value;
+            this.next = next;
+        }
+    }
 
     @Override
     public void put(K key, V value) {
+        if (size > table.length * LOAD_FACTOR) {
+            resize();
+        }
+        Node<K,V> node = table[hash(key) % table.length];
```

Comment:
You are calculating the hash and the index of the table array twice. It would be more efficient to calculate it once and store the result in a variable.